iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -62,6 +62,8 @@ public IotHubConnectionCredentials(IAuthenticationMethod authenticationMethod, s
         /// <returns>A new instance of this class.</returns>
         /// <exception cref="ArgumentNullException"><paramref name="iotHubConnectionString"/>, IoT hub host name or device Id is null.</exception>
         /// <exception cref="ArgumentException"><paramref name="iotHubConnectionString"/>, IoT hub host name or device Id are an empty string or consist only of white-space characters.</exception>
+        /// <exception cref="InvalidOperationException">Different gateway hostnames were specified through the connection string and <see cref="IotHubClientOptions.GatewayHostName"/>.
+        /// It is recommended to not hand edit connection strings but instead use <see cref="IotHubClientOptions"/> to specify values for the additional fields.</exception>
```

Comment:
```suggestion
        /// It is recommended to use <see cref="IotHubClientOptions"/> to specify values for the additional fields rather than hand edit connection strings.</exception>
```

Reply:
Same for duplicated text in the other files.